### ISSUE TITLE
NVDA cannot report position on page is MS Word

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3707]
**Reported by andrewd on 2013-12-10 03:05**
There are times when it is important to know where on the page in a word processor file one is.  That is, x centimetres from the left and y centimetres from the top.  Unless I have missed it, Word does not provide this information succinctly and it would be a valuable addition to NVDA.

**Comment by Qchristensen** [comment]
I still agree with adding this functionality, but a partial workaround for anyone searching for a way to do this in the meantime, you can add "vertical page position" to the status bar in Word (though not horizontal page position).

To show the vertical position on the status bar (this works in Word 2016 and may differ in earlier versions):
1. Ensure you are in Word
2. Press F6 to move to the status bar
3. Press the applications key to open the context menu
4. Press V to move to "Vertical page position".
5. Press ENTER to check this (note that if it is already checked, this will uncheck it).
6. Press the arrow keys to move around the status bar to read the options.
7. Press escape to move back to your document.
8. To read the status bar, press NVDA+end (in laptop layout, press NVDA+SHIFT+END).

**Comment by feerrenrut** [comment]
This is a tricky one. From a technical stand point, we might be able to do something similar to how we calculate (guess at) the current column number. There is some related discussion on the following pull request: https://github.com/nvaccess/nvda/pull/6498

**Comment by andrew-l-d** [comment]
Including the vertical position on the status bar is a big help.  Ideally, an NVDA hotkey would announce both vertical and horizontal positions.

**Comment by Adriani90** [comment]
This issue is fixed in 2018.4
cc: @Qchristensen

**Comment by ehollig** [comment]
Fixed in #9008. Closing
